code/modules/ruins/objects_and_mobs/ash_walker_den.dm

```diff
@@ -7,8 +7,8 @@
 	icon_state = "ash_walker_nest"
 	icon_living = "ash_walker_nest"
 	faction = list("ashwalker")
-	health = 200
-	maxHealth = 200
+	health = 400
+	maxHealth = 400
```

Comment:
\> play it
\> killed bubblegun as an ash walker on tg

btfo kid, this buff is not needed